<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastMinuteOffers</class>
 <widget class="QDialog" name="LastMinuteOffers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>998</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="LastMinute" name="LMOff1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>10</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Ravie</family>
             <pointsize>36</pointsize>
            </font>
           </property>
           <property name="text">
            <string>PRAGA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LMOff1DATE">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Ravie</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>Do: 01/01/2021</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="LastMinute" name="NONE_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Ravie</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INNE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NONE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Ravie</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:black;</string>
         </property>
         <property name="text">
          <string>Do: 01/01/2021</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Return">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Img/Img/tło1.jpg);</string>
     </property>
     <property name="text">
      <string>Powrót</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LastMinute</class>
   <extends>QPushButton</extends>
   <header>OfferObjects.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
